README.rst

```diff
@@ -615,6 +615,11 @@ Choose from the list of available fixers:
                         closing semicolon are
                         prohibited.
 
+* **native_function_casing** [contrib]
```

Comment:
Sorry all, I missed the comments after the initial request for the tests.
The last case from @gharlan (https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1659#discussion_r49101273) will indeed be changed by the fixer. 
If it should be changed or not is up for discussion, however changing it will not break the code (i.e. change behavior). 
For example:

```php
<?php
namespace Bar {
    function strLen($str) {
        return 'overriden' . \strlen($str);
    }
}
```

Calling:

```php
namespace {
    echo \Bar\STRLEN('xxx');
}
```

```php
namespace {
    echo \Bar\strlen('xxx');
}
```

```php
namespace {
    echo \Bar\strLEN('xxx');
}
```

Or:

```php
namespace Bar{
    echo strlen('xxx');
}
```

```php
namespace Bar{
    echo STRLEN('xxx');
}
```

```php
namespace Bar{
    echo strLEn('xxx');
}
```

will all result in the same code, so not risky.

Let me check if this is touched by the fixer:
```php
namespace {
    echo \Bar\STRLEN('xxx');
}
```

the other cases like;

```php
namespace Bar{
    echo strLEn('xxx');
}
```

cannot be tested against by the fixer as it would require knowledge of the whole code base.